### PR TITLE
{bio}[intel/2016b] SimVascular v2.16.0406

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b7f74bdb541a69980266201f1267972f for a full test report.

File: easybuild/easyconfigs/s/SimVascular/SimVascular-2.16.0406-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+name = 'SimVascular'
+version = '2.16.0406'
+
+homepage = 'http://simvascular.github.io/'
+description = """SimVascular is an open source software suite for cardiovascular simulation, providing a complete
+ pipeline from medical image data to 3D model construction, meshing, and blood flow simulation."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+# downlaod via https://simtk.org/project/xml/downloads.xml?group_id=188
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SimVascular/SimVascular-2.16.0406-intel-2016b.eb]: `:setlocal spell`

File: easybuild/easyconfigs/s/SimVascular/SimVascular-2.16.0406-intel-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+sources = ['SV-linux-x64-2016-04-06.tar.gz']
+
+dependencies = [
+    ('X11', '20160819'),
+    ('Tcl', '8.6.5'),
+    ('Tk', '8.6.5', '-no-X11'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SimVascular/SimVascular-2.16.0406-intel-2016b.eb]: why `-no-X11`? It doesn't work and you have X as a dep.

File: easybuild/easyconfigs/s/SimVascular/SimVascular-2.16.0406-intel-2016b.eb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "mkdir %(installdir)s/bin",
+    "ln -s %(installdir)s/Bin/simvascular %(installdir)s/bin/simvascular",
+    "ln -s %(installdir)s/Bin/svpost %(installdir)s/bin/svpost",
+    "ln -s %(installdir)s/Bin/svpre %(installdir)s/bin/svpre",
+    "ln -s %(installdir)s/Bin/svsolver %(installdir)s/bin/svsolver",
+]
+
+sanity_check_paths = {
+    'files': ['bin/simvascular', 'bin/svpost', 'bin/svpre', 'bin/svsolver'],
+    'dirs': ['Bin', 'lib'],
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SimVascular/SimVascular-2.16.0406-intel-2016b.eb]: is there any point in testing for `bin` when you already test for files inside `bin`?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a0543f5c1b2772b2e829ff12d37fbfaf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2017.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c0e9823037b0ea2ef07e608acece9c72 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8ddf6da13ac2d4472241b0da633d61ae for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**